final_task/rss_reader/argparse_handler.py

```diff
@@ -125,12 +126,41 @@ def valid_date(date: str) -> datetime.datetime:
         raise custom_error.NotValidDateError(msg)
 
 
+def valid_directory_path(path: str, key: str) -> str:
+    """Checks if path is valid"""
+    if path[-1] != "\\":
+        path += "\\"
+
+    if os.path.isdir(path):
+        if key == 'html':
+            filename = 'NEWS.html'
```

Comment:
It is better to have more informative name than this one. For example when it was created(preferable) and from which source news came (optional).